`DATE` is not supported

```
FAILED tests/test_ingest.py::TestIngest::test_create[trino:4nn:ingest/date] - AssertionError: Tables do not match! Diff:
--- expected
+++ actual
@@ -1,6 +1,12 @@
 idx: int64
+  -- field metadata --
+  sql.column_name: 'idx'
+  sql.database_type_name: 'BIGINT'
 value: date32[day]
-{'idx': 0, 'value': datetime.date(1970, 1, 1)}
-{'idx': 1, 'value': datetime.date(2023, 5, 15)}
-{'idx': 2, 'value': datetime.date(1960, 1, 2)}
-{'idx': 3, 'value': datetime.date(2170, 1, 2)}
+  -- field metadata --
+  sql.column_name: 'value'
+  sql.database_type_name: 'DATE'
+{'idx': 0, 'value': datetime.date(1969, 12, 31)}
+{'idx': 1, 'value': datetime.date(2023, 5, 14)}
+{'idx': 2, 'value': datetime.date(1960, 1, 1)}
+{'idx': 3, 'value': datetime.date(2170, 1, 1)}
FAILED tests/test_query.py::TestQuery::test_query[trino:4nn:type/bind/date] - OverflowError: date value out of range
FAILED tests/test_query.py::TestQuery::test_query[trino:4nn:type/literal/date] - AssertionError: Tables do not match! Diff:
--- expected
+++ actual
@@ -1,2 +1,5 @@
 res: date32[day]
-{'res': datetime.date(2023, 5, 15)}
+  -- field metadata --
+  sql.column_name: 'res'
+  sql.database_type_name: 'DATE'
+{'res': datetime.date(2023, 5, 14)}
FAILED tests/test_query.py::TestQuery::test_query[trino:4nn:type/select/date] - OverflowError: date value out of range
```

--- a/trino.go
+++ b/trino.go
@@ -106,8 +106,6 @@ func (m *trinoTypeConverter) CreateInserter(field *arrow.Field, builder array.Bu
 		return &trinoBinaryInserter{builder: builder.(array.BinaryLikeBuilder)}, nil
 	case *arrow.Date32Type:
 		return &date32Inserter{builder: builder.(*array.Date32Builder)}, nil
-	case *arrow.Date64Type:
-		return &date64Inserter{builder: builder.(*array.Date64Builder)}, nil
 	default:
 		// For all other types, use default inserter
 		return m.DefaultTypeConverter.CreateInserter(field, builder)
@@ -208,35 +206,6 @@ func (ins *date32Inserter) AppendValue(sqlValue any) error {
 	year, month, day := t.Date()
 	utcDate := time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
 	val := arrow.Date32FromTime(utcDate)
-
-	ins.builder.Append(val)
-	return nil
-}
-
-type date64Inserter struct {
-	builder *array.Date64Builder
-}
-
-func (ins *date64Inserter) AppendValue(sqlValue any) error {
-	unwrapped, err := unwrap(sqlValue)
-	if err != nil {
-		return err
-	}
-	if unwrapped == nil {
-		ins.builder.AppendNull()
-		return nil
-	}
-
-	t, ok := unwrapped.(time.Time)
-	if !ok {
-		return fmt.Errorf("expected time.Time for date64 inserter, got %T", sqlValue)
-	}
-
-	// Convert to date without timezone conversion
-	// Extract just the date components and calculate days since epoch manually
-	year, month, day := t.Date()
-	utcDate := time.Date(year, month, day, 0, 0, 0, 0, time.UTC)
-	val := arrow.Date64FromTime(utcDate)
 
 	ins.builder.Append(val)
 	return nil

